assets/css/main.css

```diff
@@ -1,3 +1,7 @@
+#query-monitor-main .ep-queries-debug.wpd-queries {
+	padding: 10px !important;
```

Comment:
Any way we can avoid the `!important`? Add an extra ID to the selector or something?